### PR TITLE
CPDTP-170 Update API Sandbox landing page content

### PR DESCRIPTION
**Pull request by knowdigital**
Have added the latest copy as at 21 May 14.53 
WIll remove comments once all copy signed off

### Context

https://dfedigital.atlassian.net/browse/CPDTP-170

### Review Checks
- [ ] All pages have automated accessibility checks via cypress
- [ ] All pages have visual tests via cypress + percy
- [ ] All `School` queries are correctly scoped - `eligible` when they need to be

### CODE REVIEW

**Comment by kicferk1** [comment]
Can we give @knowdigital access to our repo, so that our review apps work on his PRs? @jamesrose I think you went through that at least once already :)

File: app/views/sandbox/show.html.erb [hunk 1]
```diff
@@ -1,16 +1,45 @@
...
-      <li>Authenticate</li>
-      <li>Test the API <%= govuk_link_to "API integrations", "/api-docs" %></li>
-    </ul>
 
-  </div>
+
+            <h2 class="govuk-heading-m">Help and support</h2>
+                <p>Email us at <a href="mailto:becomingateacher@digital.education.gov.uk">becomingateacher@digital.education.gov.</a> for support using this sandbox.</p>
+
+    <!--         <p>When Apply receives an application, it can then be seen by others</p>
```
Comment (timabell) [inline app/views/sandbox/show.html.erb:29]: remove the commented out code please

**Review by timabell — CHANGES_REQUESTED** [review]

File: app/views/sandbox/show.html.erb [hunk 2]
```diff
@@ -1,16 +1,45 @@
...
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">Use our sandbox to test Early Career Framework</h1>
+            <p>This is an API test environment which lets you simulate the Early Career Framework service as a training provider.</p>
+            <p>You can:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>sign in as a training provider and confirm the start of an applicant's course</li>
+             <!--    <li>sign in as a training provider and process an applicant</li>
+                <li>test the system’s API capabilities</li> -->
+            </ul>
+            <p>You can give us feedback at any time to help us improve the service</p>
```
Comment (blackrat) [inline app/views/sandbox/show.html.erb:14]: Missing fullstop at the end of this line. We should really be consistent outside of bullet points.

File: app/views/sandbox/show.html.erb [hunk 3]
```diff
@@ -1,16 +1,45 @@
...
+            <h2 class="govuk-heading-m">Get started</h2>
+            <p> The ECF service allows you to integrate your software with ours and test it.</p>
```
Comment (blackrat) [inline app/views/sandbox/show.html.erb:16]: Extra space after the opening <p>

Comment (blackrat) [reply app/views/sandbox/show.html.erb:16]: Not really a big deal for html parsing, but for someone looking at the source, it's obvious

**Review by blackrat — COMMENTED** [review]
Just a couple of stylistic things. Content seems fine

File: app/views/sandbox/show.html.erb (hunk 2 above)
Comment (knowdigital) [reply app/views/sandbox/show.html.erb:14]: ok cool, will update

File: app/views/sandbox/show.html.erb [hunk 4]
```diff
@@ -1,16 +1,45 @@
...
-    <h1 class="govuk-heading-xl"><%= t('page_titles.sandbox') %></h1>
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">Use our sandbox to test Early Career Framework</h1>
+            <p>This is an API test environment which lets you simulate the Early Career Framework service as a training provider.</p>
+            <p>You can:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>sign in as a training provider and confirm the start of an applicant's course</li>
+             <!--    <li>sign in as a training provider and process an applicant</li>
```
Comment (blackrat) [inline app/views/sandbox/show.html.erb:11]: Better to remove rather than to commented out markup.

**Comment by timabell** [comment]
@knowdigital sorry, could you re-raise this as a new PR now you have team access. The review apps don't work with PRs from forks so we can't merge this one.

**Pull request closed without merging**